stable/logspout/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A Helm chart for logspout log forwarder
+name: logspout
+version: 0.1.0
```

Comment:
Add `appVersion`

Reply:
@unguiculus, Thanks for your review, I've made the suggested edits.

Reply:
Please follow my comment about namespacing.